lib/xcode/install.rb

```diff
@@ -49,8 +49,8 @@ def current_symlink
     end
 
     def download(version, progress, url = nil)
-      return unless exist?(version) || url
-      xcode = seedlist.find { |x| x.name == version }
+      return unless url || exist?(version)
+      xcode = seedlist.find { |x| x.name == version } unless url
```

Comment:
The previous line won't return if the `url` is non-nil, and won't call `exist?` either. We don't want to be interacting with Apple in any way if we provide our own URL, and that includes getting the seedlist.